frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -0,0 +1,693 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import CatalogTileView from 'patternfly-react-extensions/dist/js/components/CatalogTileView/CatalogTileView';
```

Comment:
Tests pass using `import { CatalogTileView } from 'patternfly-react-extensions'`.

Reply:
This will likely pull in a lot more code than required and increase the chunk size.

Reply:
Pulling in index.js at the top level references all components so they won't get shaked. The suggested use is to include only what is used rather than pulling from the top level.

Reply:
OK, I'm mistaken. I made the change and the size of the chunk actually decreased 😕

Reply:
Webpack is smart enough to automatically tree-shake if using ES6 module syntax!